Cargo.toml

```diff
@@ -25,6 +25,12 @@ failure = "0.1.1"
 os_type = "2.0.0"
 backtrace = "0.3"
 
+[target.'cfg(target_os = "linux")'.dependencies]
+x11-dl = "2"
+
+[target.'cfg(target_os = "windows")'.dependencies]
+winapi = { version = "0.3", features = ["winuser"] }
+
 [features]
```

Comment:
I would absolutely want to make either of these an optional dependency. We can't have CLI applications that run headless depend on X11. Adding a feature called `gui` (or something along those lines :bike: :derelict_house: ...) will make the feature opt-in which is a much safer option here

Reply:
I have added winapi and x11-dl to the gui feature